src/sweets/cli.py

```diff
@@ -82,7 +82,7 @@ def _get_cli_args() -> dict:
         ),
     )
     aoi.add_argument(
-        "--data-dir",
+        "--out-dir",
```

Comment:
Hmmm, did this fix it for you? I see it's got 
```python
download_orbits(self.asf_query.out_dir, self.orbit_dir)
```
, so I wouldn't think that `out_dir` as a keyword arg would populate that correctly